MachObfuscator/Commons/Paths.swift

```diff
@@ -1,6 +1,23 @@
+import Foundation
+
 enum Paths {
     static let separator: Character = "/"
-    static let iosRuntimeRoot = "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot"
     static let macosFrameworksRoot = "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk"
     static let iosFrameworksRoot = "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk"
+
+    static let iosRuntimeRoot: String = {
```

Comment:
After this change `test_shouldCollectExternalDependencies_AsUnobfuscable_AndAppendRuntimePrefix_WhenIOSPlatform` breaks. Fix:

```patch
diff --git a/MachObfuscatorTests/DependencyAnalysisTests/ObfuscationPaths+Building_forAllExecutables_Tests.swift b/MachObfuscatorTests/DependencyAnalysisTests/ObfuscationPaths+Building_forAllExecutables_Tests.swift
index 9e6705cd4fe84bfe3493c8ff068787020e7615ed..6b5acc924ec40ac0cf8f71b414ce278639a05c9e 100644
--- a/MachObfuscatorTests/DependencyAnalysisTests/ObfuscationPaths+Building_forAllExecutables_Tests.swift
+++ b/MachObfuscatorTests/DependencyAnalysisTests/ObfuscationPaths+Building_forAllExecutables_Tests.swift
@@ -313,13 +313,12 @@ class ObfuscationPaths_Building_forAllExecutables_Tests: XCTestCase {
                                     isExecutable: false,
                                     dylibs: [externalDependency2DylibEntry])
 
-        let runtimePrefixedExternalDependency1Path = "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libobjc.A.dylib"
+        let runtimePrefixedExternalDependency1Path = Paths.iosRuntimeRoot + "/usr/lib/libobjc.A.dylib"
         testRepository.addMachOPath(runtimePrefixedExternalDependency1Path, platform: .ios, isExecutable: false)
 
-        let runtimePrefixedExternalDependency2Path = "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/UIKit.framework/UIKit"
+        let runtimePrefixedExternalDependency2Path = Paths.iosRuntimeRoot + "/System/Library/Frameworks/UIKit.framework/UIKit"
         testRepository.addMachOPath(runtimePrefixedExternalDependency2Path, platform: .ios, isExecutable: false)
-        let runtimePrefixedExternalLibrary =
-            "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/MockSystem/Library/Frameworks/AdSupport.framework/AdSupport"
+        let runtimePrefixedExternalLibrary = Paths.iosRuntimeRoot + "/MockSystem/Library/Frameworks/AdSupport.framework/AdSupport"
         testRepository.addMachOPath(runtimePrefixedExternalLibrary, platform: .ios, isExecutable: false)
 
         let externalFrameworkSDKPath = Paths.iosFrameworksRoot + externalFramework

```

Reply:
Thanks, will apply that right away!